### PR TITLE
Update background.md

### PR DESCRIPTION
**Pull request by brishtibheja**
I have simplified the language, changed the formatting in list (to what we have in the landing page), removed references to words that everyone might not understand (Latex).

### CODE REVIEW

File: src/background.md [hunk 1]
```diff
@@ -9,30 +9,29 @@ amount you learn.
...
-been written about in the scientific literature for many years.
-Understanding how they work will make you a more effective learner.
+**spaced repetition**. They are not known to most learners, despite being well-documented in scientific literature.
+Understanding how these concepts work will make you a more effective learner.
 
 ## Active Recall Testing
 
 _Active recall testing_ means being asked a question and trying to
-remember the answer. This is in contrast to _passive study_, where we
+remember the answer. This is in contrast to _passive study_, where we
```
Comment (dae) [inline src/background.md:34]: Accident?

File: src/background.md [hunk 2]
```diff
@@ -112,10 +111,10 @@ In SuperMemo's spaced repetition system, every time you answer a
 question, you tell the program how well you were able to remember it —
 whether you forgot completely, made a small mistake, remembered with
 trouble, remembered easily, etc. The program uses this feedback to
-decide the optimal time to show you the question again. Since a memory
+decide the optimal time to show you the question again. Because a memory
 gets stronger each time you successfully recall it, the time between
-reviews gets bigger and bigger — so you may see a question for the first
-time, then 3 days later, 15 days later, 45 days later, and so on.
+reviews gets larger and larger — so you may see a question for the first
```
Comment (dae) [inline src/background.md:116]: longer?

**Review by dae — COMMENTED** [review]
A couple of minor thoughts:

File: src/background.md (hunk 1 above)
Comment (brishtibheja) [reply src/background.md:34]: Oh, yes. Is that a problem? I didn't bothered to undo that.

File: changed larger to longer [hunk 3]
```diff
@@ -9,30 +9,29 @@ amount you learn.
...
-been written about in the scientific literature for many years.
-Understanding how they work will make you a more effective learner.
+**spaced repetition**. They are not known to most learners, despite being well-documented in scientific literature.
+Understanding how these concepts work will make you a more effective learner.
 
 ## Active Recall Testing
 
 _Active recall testing_ means being asked a question and trying to
-remember the answer. This is in contrast to _passive study_, where we
+remember the answer. This is in contrast to _passive study_, where we
```
Comment (dae) [inline changed larger to longer:34]:
```suggestion
remember the answer. This is in contrast to _passive study_, where we
```

Trailing spaces can have a different meaning in markdown; better not to change it.

**Pull request merged**